packages/workers-shared/router-worker/src/worker.ts

```diff
@@ -77,6 +84,48 @@ export default {
 
 			const maybeSecondRequest = request.clone();
 
+			let asset: "static_routing" | "ignored" | true | false = false;
```

Comment:
I don't _love_ mutable variables like this (though absolutely have used them in a few places myself!)

Is it at all cleaner if this is becomes an arg to the `routeToUserWorker` and `routeToAssets` functions?

Reply:
cleaner is relative here IMO. the typing becomes cleaner but the call sites is more verbose. no strong opinion here